examples/autokey_setup/main.tf

```
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "autokey" {
+  source = "../../modules/autokey"
```

Per best practices, the examples should demonstrate using the registry:

```suggestion
  source  = "terraform-google-modules/terraform-google-kms/google//modules/autokey"
  version = "~> 4.0"
```

The CI will automatically swap to `source = "../../modules/autokey"` during Int and Lint testing, so you don't have to worry about the version string.

---

done.

---

wanted to demonstrate with registry but thought of merging this cl first, as we will get the latest version with new change after merge.

Then use the generated version for autokey submodule and source as tf registry in example.